src/advanced-calculator/calculator.module.css

```diff
@@ -97,12 +102,16 @@ input[type='number'] {
   justify-content: center;
   height: 100%;
   width: 100%;
-  position: sticky;
   top: 0;
   grid-column: 2 / 3;
   grid-row: span 1;
   padding: 1.5rem;
 }
+@media (min-width: 900px) {
+  .calculator-controls {
+    position: sticky;
```

Comment:
Strictly speaking not an a11y change IMO. Can do this change but then I'd argue that we can remove the show/hide toggle also as it does nothing.

Reply:
[This is an a11y issue](https://www.uutilsynet.no/wcag-standarden/134-visningsretning-niva-aa/141). Not current law, and won't apply to us as a private company (yet). But still have effect on real people.
Can agree the solution could be better but that would have to be figured out.

Reply:
To be clearer, as mentioned in a previous comment. A lot of what we do here is trying to clearly show potential salery and in some cases comparing numbers. And on phones this is if I recall correctly the only place it is shown. So with this change we scroll down and the essential information is no longer easily accessible. My opinion is that that has more negative impact in the short run than taking too much space in landscape. As it affects more users. Non is ideal ofc. Fixing the underlying issue with the design (or extending to be more sophisticated with in/out by scroll etc) is best but probably more time consuming. My opinion is that until that is prioritized the current non-ideal affects less people than this proposed non-ideal.

Reply:
Dropped for now. Should I create an Issue or how should we address this?

Reply:
Might also have @vegardfeste for instance look at it to see if there's a quick win here, but at least solve it in a separate issue so we can merge this.

Reply:
Everything should be good, will you resolve comments